.gitlab-ci.yml

```diff
@@ -567,9 +569,16 @@ build-rust-doc:
     - buildah push --format=v2s2 "$IMAGE_NAME:latest"
   after_script:
     - buildah logout "$IMAGE_NAME"
-    # pass artifacts to the trigger-simnet job
+    # pass artifacts to the trigger-simnet job 
+    - echo "IMAGE_NAME is ${IMAGE_NAME}"
     - echo "IMAGE_NAME=${IMAGE_NAME}" > ./artifacts/$PRODUCT/build.env
-    - echo "IMAGE_TAG=${VERSION}" >> ./artifacts/$PRODUCT/build.env
+    - ls -la
+    - VERSION="$(cat ./artifacts/$PRODUCT/VERSION)"
+    - echo "VERSION is ${VERSION}"
+    - IMAGE_TAG="${VERSION}"
+    - echo "IMAGE_TAG is ${IMAGE_TAG}"
+    - echo "IMAGE_TAG=${IMAGE_TAG}" >> ./artifacts/$PRODUCT/build.env
+    - cat ./artifacts/$PRODUCT/build.env
```

Comment:
please remove the unneeded debug info like `ls -la` and echoing the same value 2 times.

Reply:
@radupopa2010 piped `tee` does literally the same what `cat` does, why did you leave it here?